packages/flutter_test/lib/src/image.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:typed_data';
+import 'dart:ui' as ui;
+
+import 'package:flutter/painting.dart';
+
+import 'test_async_utils.dart';
+
+final Map<int, ui.Image> _cache = <int, ui.Image>{};
+
+/// Asynchronously creates an arbitrarily sized test image.
```

Comment:
Maybe instead of "arbitrarily sized test image" -> "arbitrarily sized image for testing", since it is just a regular image.

I dunno if async is necessarily important here, since that is implied by the return type.

Reply:
Done.

Reply:
(Cleaned up a little of the rest of the doc too, and saved a file I had fixed but forogtten to save :\)